### ISSUE TITLE
Transfer submission files (script, JDF) to dCache at submission time?

### ISSUE THREAD

**Issue by retzkek** [marcmengel/jobsub_lite#311]
This is for discussion: right now we have to transfer the description file to the job, in order for it to get copied out to dCache. We could copy it and the script there right at submission time from jobsub_submit.

Pros:
* creates output destination, users won't get 404 error trying to see what files are there (currently it won't be created until the first job has finished)
* don't have to copy the JDF in input_files

Cons:
* more complexity in jobsub_submit, including IFDH (?) dependency... or maybe we just do straight webDAV calls since it's a very well-defined transfer?

**Comment by marcmengel** [comment]
So we have a "cp" in fake_ifdh.py already, it just directly calls gfal-copy. (already used for pnfs tarfile caching).

**Comment by retzkek** [comment]
I think we need this sooner than later, jobs are still getting held from failing to copyback "simple.sh" so let's just get rid of it.

**Comment by shreyb** [comment]
Yeah - let's do it.  We can even sneak in a 1.1.3 release if nothing else is ready in a few days.
